packages/angular/test/base/src/app/lazy/inputs/inputs.component.html

```diff
@@ -71,6 +71,16 @@
       <ion-note slot="end">{{input}}</ion-note>
     </ion-item>
 
+    <ion-item>
+      <ion-input-otp [(ngModel)]="inputOtp">Input OTP</ion-input-otp>
+      <ion-note slot="end">{{inputOtp}}</ion-note>
+    </ion-item>
+
+    <ion-item color="dark">
+      <ion-input-otp [(ngModel)]="inputOtp">Input OTP Mirror</ion-input-otp>
+      <ion-note slot="end">{{inputOtp}}</ion-note>
+    </ion-item>
```

Comment:
The reset button doesn't seem to do anything with these otp fields. I would expect the fields to be cleared.

Reply:
Great catch! This was a bug in how the values were being handled. Added a fix & tests here: https://github.com/ionic-team/ionic-framework/pull/30386/commits/4015ee4fea8cb6d6b17ef935aecfb7919137a09d